presto-docs/src/main/sphinx/release.rst

```diff
@@ -5,6 +5,7 @@ Release Notes
 .. toctree::
     :maxdepth: 1
 
+    release/release-0.278
```

Comment:
@wanglinsong, the given script actually created 0.278 files instead 0.279, do you know if we are missing anything here?